lcls_tools/superconducting/scLinac.py

```diff
@@ -197,11 +223,19 @@ def issueMoveCommand(self, numSteps):
         print("Waiting 5s for the motor to start moving")
         sleep(5)
         
-        while caget(self.motor_moving_pv.pvname) == 1:
+        while self.motor_moving_pv.value == 1:
+            if self.abort_flag:
+                self.abort_pv.put(1)
+                raise utils.StepperAbortError(
+                        f"Abort Requested for {self.cavity.cryomodule.name} cavity {self.cavity.number} stepper tuner")
+            
             print("Motor moving", datetime.now())
             sleep(1)
+            if (abs(self.cavity.detune_best_PV.value) > 150000
+                    and self.cavity.freq_stop_pv.value != 400000):
+                self.cavity.set_chirp_range(400000)
         
-        if caget(self.motor_done_pv.pvname) != 1:
+        if self.motor_done_pv.value != 1:
             raise utils.StepperError("Motor not in expected state")
```

Comment:
Would it make sense to include the CM/cavity number in this message?

Reply:
Probably